app/frontend/components/domains/requirements-library/requirement-field-display/generic-multi-display.tsx

```diff
@@ -0,0 +1,114 @@
+import { Box, BoxProps, Button, Heading, HeadingProps } from "@chakra-ui/react"
+import { Plus } from "@phosphor-icons/react"
+import React from "react"
+import { FieldValues } from "react-hook-form"
+import { useTranslation } from "react-i18next"
+import { getRequirementTypeLabel } from "../../../../constants"
+import { ERequirementType } from "../../../../types/enums"
+import { TEditableHelperTextProps } from "../requirement-field-edit/editable-helper-text"
+import { TEditableLabelProps } from "../requirement-field-edit/editable-label"
+import { GenericFieldItemDisplay } from "./generic-field-item-display"
+import { TRequirementFieldDisplayProps } from "./index"
+
+export type TGenericDisplayProps<TFieldValues extends FieldValues> = {
+  fieldItems: Array<{
+    type: ERequirementType
+    key: string
+    label: string
+    required?: boolean
+    containerProps?: BoxProps
+  }>
+  containerProps?: BoxProps
+  editableInput?: JSX.Element
+  editableLabelProps?: TEditableLabelProps<TFieldValues>
+  editableHelperTextProps?: TEditableHelperTextProps<TFieldValues>
+  addAnotherText?: string
+  renderHeading?: () => JSX.Element
+} & TRequirementFieldDisplayProps
+
+export function GenericMultiDisplay<TFieldValues>({
+  fieldItems,
+  label,
+  labelProps,
+  showAddLabelIndicator,
+  renderHeading,
+  containerProps,
+  showAddPersonButton,
```

Comment:
if this component is generic, this should probably be renamed to showAddAnotherButton